package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.0.0",
   "main": "main.js",
   "scripts": {
-    "electron": "ng build --base-href ./ && electron .",
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng build --base-href ./ && electron .",
+    "start:angular": "ng serve",
```

Comment:
@CubikNeRubik it wont work if you run app with `ng serve`
for unknown reason build just fails

Reply:
@CubikNeRubik @EugeneTi 
ok, merge it anyway
i found a problem that dont allow to start app with `ng serve`
will try to fix it later